docs/en/core/writing-modes.md

```diff
@@ -0,0 +1,16 @@
+# Writing Modes
+In the View Menu, Zettlr features several Editor settings that may enhance your writing experience. Have a play and see what works for you!
+
+## Night Mode
+Also commonly known as 'Dark Mode', clicking this option will change the background colour of the Editor to black and the default text colour to white.
```

Comment:
We should call that Dark mode consistently, I have to fix that in the translations sometime (very) soon!

Reply:
Should change this in docs after you change in the app?

Reply:
Fixed and renamed to Editor modes